big_tests/tests/gdpr_SUITE.erl

```diff
@@ -585,6 +660,75 @@ retrieve_multiple_private_xmls(Config) ->
               Alice, Config, "private", ExpectedHeader, ExpectedItems)
         end).
 
+retrieve_inbox_muclight(Config) ->
+    escalus:fresh_story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
+        muc_light_helper:given_muc_light_room(?ROOM, Alice, [{Bob, member}]),
+
+        Body = <<"Are you sure?">>,
+        Res = muc_light_helper:when_muc_light_message_is_sent(Alice, ?ROOM, Body, <<"9128">>),
+        muc_light_helper:then_muc_light_message_is_received_by([Alice, Bob], Res),
+        ExpectedHeader = ["jid", "content", "unread_count", "timestamp"],
+        ExpectedAliceItems = [
+                              #{
+          "jid" => [{contains, <<"muclight.localhost">>},
+                    {contains, ?ROOM}],
```

Comment:
We have helper for retrieving muclight domain, and moreover I think the Room name comes with the domain so no need for separate `contains` validators. I mean something like this (didnt check if it compiles):
```
Domain = muclight_domain(),
{contains, <<?ROOM/binary,"@', Domain/binary>>
```